### DISCUSSION TITLE
How to create a smooth gradual gradient from white to black

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#6603] [Help] [answered]
### ImageMagick version

7.1.0-62-Q16

### Operating system, version and so on

Windows 10

### Description

Hi. How can I create the smoothest gradient from white to black? It does not have to be mathematically correct, but I needs be visually smooth so it is as unnoticeable as possible to spot the transition from white to black all the way though. Any advice is appreciated!

**Comment by fmw42** [comment]
How big a gradient?  Gradient: goes from 0 to 255, so 256 samples is needed to make it smooth.  How big of an image do you need?  Basic gradient is

`magick -size 256x256 gradient: gradient.png`

Change the height to make the gradient shorter or longer, but still having values black (0) and white (255).

**Reply by ghost** [reply]
The size is 1024x1024. Okay, I will give that a try and adjust the height accordingly. Thank you.

**Reply by fmw42** [reply, marked as answer]
Replace 256x256 with 1024x1024.
